include/zephyr/xen/public/version.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: MIT */
+/******************************************************************************
+ * version.h
+ *
+ * Xen version, type, and compile information.
+ *
+ * Copyright (c) 2005, Nguyen Anh Quynh <aquynh@gmail.com>
+ * Copyright (c) 2005, Keir Fraser <keir@xensource.com>
+ */
+
+#ifndef __XEN_PUBLIC_VERSION_H__
+#define __XEN_PUBLIC_VERSION_H__
+
+#if !defined(CONFIG_XEN_INITIAL_DOMAIN)
```

Comment:
(note) you can use `#ifndef` instead `#if !defined`

Reply:
I prefer to use `#if !defined`.

Reply:
than why have you used ifndef a few lines above in this file?

Reply:
1. `#ifndef` and `#define` have same length
2. That is template

Reply:
ok, in any case for me it is only some kind of cosmetic changes
so, I'm not against to leave it as is